src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -12,8 +12,13 @@
 NODE_RUNTIME_NAME = "node"
 PYTHON_RUNTIME_NAME = "python"
 OS_DEFAULT = "Windows"
-STATIC_RUNTIME_NAME = "static"  # not an oficial supported runtime but used for CLI logic
+STATIC_RUNTIME_NAME = "static"  # not an official supported runtime but used for CLI logic
 NODE_VERSIONS = ['4.4', '4.5', '6.2', '6.6', '6.9', '6.11', '8.0', '8.1', '8.9', '8.11']
 NETCORE_VERSIONS = ['1.0', '1.1', '2.0']
 DOTNET_VERSIONS = ['3.5', '4.7']
 LINUX_SKU_DEFAULT = "P1V2"
+RUNTIME_TO_IMAGE = {
+    'node': 'mcr.microsoft.com/azure-functions/node:2.0',
+    'dotnet': 'mcr.microsoft.com/azure-functions/dotnet-core:2.0',
```

Comment:
the image name should be `mcr.microsoft.com/azure-functions/dotnet:2.0`

Reply:
Oh, right! Thanks @ahmelsayed. I confused this with `microsoft/azure-functions-dotnet-core2.0:2.0`. Fixed now.